fe/fe-core/src/main/java/org/apache/doris/statistics/util/StatisticsUtil.java

```diff
@@ -906,6 +906,16 @@ public static long getHugeTableAutoAnalyzeIntervalInMillis() {
         return StatisticConstants.HUGE_TABLE_AUTO_ANALYZE_INTERVAL_IN_MILLIS;
     }
 
+    public static long getExternalTableAutoAnalyzeIntervalInMillis() {
+        try {
+            return findConfigFromGlobalSessionVar(SessionVariable.EXTERNAL_TABLE_AUTO_ANALYZE_INTERVAL_IN_MILLIS)
+                .externalTableAutoAnalyzeIntervalInMillis;
+        } catch (Exception e) {
+            LOG.warn("Failed to get value of externalTableAutoAnalyzeIntervalInMillis, return default", e);
```

Comment:
Why is an exception thrown here?

Reply:
I didn't dive deep in this, I simply followed all the other variables' format. Seems like VariableMgr.getValue may throw exception when the session name not exist.